api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -10,10 +10,14 @@
 package org.openmrs;
 
 import java.util.Date;
+import java.util.Objects;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * @since 2.1.0
+ */
```

Comment:
I changed Patient patient to Integer patientId. However lot of tests are failing right now.

```
ERROR - BasicPropertyAccessor$BasicSetter.set(121) |2017-04-14 19:41:50,315| HHH000123: IllegalArgumentException in class: org.openmrs.CohortMembership, setter method of property: patientId
ERROR - BasicPropertyAccessor$BasicSetter.set(122) |2017-04-14 19:41:50,316| HHH000091: Expected type: java.lang.Integer, actual value: org.openmrs.Patient_$$_jvst4ea_28

IllegalArgumentException occurred while calling setter for property [org.openmrs.CohortMembership.patientId (expected type = java.lang.Integer)]; target = [CohortMembership[hashCode=c700c9ba,uuid=2bc2a1bf-bc54-410d-8721-7fb16375be91]], property value = [Patient#2] setter of org.openmrs.CohortMembership.patientId
IllegalArgumentException occurred while calling setter for property [org.openmrs.CohortMembership.patientId (expected type = java.lang.Integer)]; target = [CohortMembership[hashCode=c700c9ba,uuid=2bc2a1bf-bc54-410d-8721-7fb16375be91]], property value = [Patient#2]
```

It looks like something's a miss. Is it because of the <many-to-one> property in the hbm?